### ISSUE TITLE
Section 2.1.2 Data Types and value constraints not consistent with TD spec

### ISSUE THREAD

**Issue by takuki** [w3c/wot-binding-templates#65]
In Section 2.1.2, it says:

> It is up to the client to adapt to the range and type as well as the engineering units provided.

This is true for downstream payload (from Thing to Consumer). 

For upstream payload, on the other hand, Consumer needs to honor those constraints as much as possible.

According to section 8.2 Data Schemas of TD spec, it says:

>  In general, Consumers should follow the data schemas strictly, not generating anything not given in the WoT Thing Description, but should accept additional data from the Thing not given explicitly in the WoT Thing Description.

**Comment by egekorkan** [comment]
The intention is the same, we will rephrase it to match the TD spec

**Comment by egekorkan** [comment]
@takuki can you review it and close if it is OK

**Comment by takuki** [comment]
Looks good. Thank you.

**Comment by takuki** [comment]
Looks good. Thank you!
